Alignment/APEEstimation/python/SectorBuilder_Bpix_cff.py

```diff
@@ -35,11 +35,16 @@
     name = 'BpixLayer3',
     layer = [3],
 )
+BpixLayer4 = Bpix.clone(
+    name = 'BpixLayer4',
+    layer = [4],
+)
 
 BPIXLayerSeparation = cms.VPSet(
     BpixLayer1,
     BpixLayer2,
     BpixLayer3,
+    BpixLayer4,
```

Comment:
I added a phase0 version. Since phaseI is likely to be the default from now on this makes more sense to me than using phase0 as default and phaseI as an extra file